tests/system/providers/google/firebase/example_firestore.py

```diff
@@ -56,6 +56,7 @@
     BigQueryInsertJobOperator,
 )
 from airflow.providers.google.firebase.operators.firestore import CloudFirestoreExportDatabaseOperator
+from airflow.utils.trigger_rule import TriggerRule
 
 GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "example-gcp-project")
```

Comment:
The amount of the env variables is one of the problems with the old design. When migrating to the new design it would be best to reduce required env variables rewriting them to constants or fstring reusing ENV_ID and DAG_ID whenever needed. The example of removing the need of environment variable can be seen in this PR: https://github.com/apache/airflow/pull/23282/files#diff-8d21adbd43cba4bdb6175bd7fcc99cc7352a7f2788c9b3486a86148985950a6aL55

Reply:
And there is no GCP_PROJECT_ID - it was replaced by SYSTEM_TESTS_GCP_PROJECT